TGC/events/Filibuster.txt

```diff
@@ -0,0 +1,593 @@
+### Filabusters ###
+
+	## William Walker invades Nicaragua
+		country_event = {
+			id = 8870780
+			title = "EVTNAME8870780"
+			desc = "EVTDESC8870780"
+			picture = "Cubanrebels"
+
+			fire_only_once = yes
+			trigger = {
+				tag = NIC
+				year = 1854
+				NOT = { year = 1860 }
+				owns = 2198
+				owns = 2197
+				owns = 2199
+			}
+
+			mean_time_to_happen = {
+				days = 286
+			}
+
+			# Don't play as FIL
+			option = {
+				name = "EVTOPTA8870780"
+
+				prestige = -5
+				NIC = {
+					all_core = {
+						add_core = FIL
+					}
+					random_owned = {
+						limit = { province_id = 2198 }
+						secede_province = FIL
+					}
+					random_owned = {
+						limit = { province_id = 2197 }
+						secede_province = FIL
+					}
+				}
+
+				FIL = {
+					activate_technology = military_staff_system
+					activate_technology = iron_muzzle_loaded_artillery
+
+				}
+
+				USA = {
+					random_pop = {
+						limit = {
+							culture = dixie
+							is_primary_culture = no # Workaround to select Dixie only since has_pop_culture=dixie and culture=dixie both can select Yankee
```

Comment:
culture = <culture> is a province scope condition that asks if the province is majority some culture. has_pop_culture *should* be what you're looking for. Has your testing shown that it sometimes selects Yankee pops despite having has_pop_culture = dixie?

Reply:
Yeah has_pop_culture should be it but it can target Yankee for some reason. My solution usually works but theres a small chance it move Sioux/etc. pops.